recipes/libnetcdf/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+if [[ $(uname) == Darwin ]]; then
+  # https://www.unidata.ucar.edu/support/help/MailArchives/netcdf/msg12897.html
+  export LIBS="-lnetcdf -lhdf5 -lhdf5_hl"
+fi
+
+if [[ $(uname) == Darwin ]]; then
+  export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+elif [[ $(uname) == 'Linux' ]]; then
+  export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+fi
+
+export LDFLAGS="-L$PREFIX/lib"
```

Comment:
It's important to always prepend or append to LDFLAGS, CFLAGS, etc.  Conda-build does use the variable to define things in the build environment.
`export LDFLAGS=-L$PREFIX/lib $LDFLAGS`